src/Umbraco.Cms.ManagementApi/Filters/SystemTextJsonConfigurationAttribute.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Text.Json;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+
+namespace Umbraco.Cms.ManagementApi.Filters;
+
+public class SystemTextJsonConfigurationAttribute : TypeFilterAttribute
```

Comment:
Suggest renaming this to `ManagementApiJsonConfigurationAttribute`, to make it abundantly clear what it is used for.